homeworks/vladimir.karbivnychiy_VladimirSkarb/homework_3/rec.js

```diff
@@ -0,0 +1,29 @@
+const readline = require('readline');
+
+const inputData = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+});
+
+inputData.question('Введите минимальное значение: ', (min) => {
+    inputData.close();
+    let a = 1;
+    let b;
```

Comment:
you do not need these variables, especially out your function scope

you just need to keep dividing by 2 until the number equals/not equals 1

make it simpler